test/e2e/service_test.go

```diff
@@ -81,3 +82,60 @@ func (data *TestData) createClusterIPServiceAndBackendPods(t *testing.T, name st
 
 	return svc, cleanup
 }
+
+// TestNodePortWindows tests NodePort Service on Windows Node. It is a temporary test to replace upstream Kubernetes one:
+// https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/test/e2e/windows/service.go#L42
+func TestNodePortWindows(t *testing.T) {
```

Comment:
There should be a link to https://github.com/antrea-io/antrea/issues/2289, otherwise it is not clear why we need this test

Reply:
Updated.